tools/repl/swift/main.c

```diff
@@ -12,9 +12,10 @@
 
 #ifdef __APPLE__
 #include <mach/mach_time.h>
+#include <dlfcn.h>
 #endif
 
-#ifdef __APPLE__
+#ifdef __gnu_linux__
```

Comment:
Linux has kind aleph-zero different ways of telling to the preprocessor "am I running Linux", so we might change this in the future.

Reply:
This was the first macro I found in clang that can be used to identify linux targets.

But, yes, we'll absolutely need to rethink this when we work on Windows support.